src/StatusQ/Controls/StatusBanner.qml

```diff
@@ -1,31 +1,45 @@
 import QtQuick 2.14
 
 import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
 
 Column {
     id: statusBanner
     width: parent.width
 
-    property color backgroundColor
-    property color bordersColor
-    property color fontColor
     property string statusText
+    property int type: StatusBanner.Type.Info
     property int textPixels: 15
     property int statusBannerHeight: 38
 
+    // "private" properties
+    QtObject {
+           id: d
```

Comment:
Any reason this is called `d`? has this some sort of semantic meaning?

Reply:
Here in [Qt encoding conventions](https://doc.qt.io/archives/qt-4.8/qml-coding-conventions.html#private-properties) they propose it like this, but if you have another specific convention, I can change without problem. Share with me which ones you follow, please!

Reply:
No need to change this! Was just curious as `d` doesn't seem to have any particular meaning.